packages/medusa/src/migrations/1698160215000-cart-sales-channels-link.ts

```diff
@@ -0,0 +1,42 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+import SalesChannelFeatureFlag from "../loaders/feature-flags/sales-channels"
```

Comment:
question: Would it make sense to merge the migrations?

Reply:
Yep, just wanted to separate files by domain but it can be all together.